ckanext/dataset_subscriptions/tests/actions/test_whatsapp_notifications.py

```diff
@@ -86,24 +86,20 @@ def create_user_with_resources(with_activity, with_notifications_enabled):
 
 @pytest.mark.usefixtures("clean_db")
 @pytest.mark.usefixtures("with_plugins")
-def test_if_whatsapp_notifications_disabled():
-    user = create_user_with_resources(with_activity=True, with_notifications_enabled=False)
-    if whatsapp_notifications.whatsapp_notifications_enabled(user):
-        phonenumber = whatsapp_notifications.get_phonenumber(user)
-    else:
-        phonenumber = None
-    assert phonenumber is None
+@pytest.mark.parametrize("activity,notifications,expected_phonenumber", [(True, False, "+1234"), (True, True, "+1234")])
```

Comment:
you only have one actual parameter here - `notification` (better rename it to `notifications_enabled`).
The rest of the params are constant and doesn't change value so just inline them in the test code.

The same applies to the other test `test_whatsapp_notifications_disabled_enabled`